benchmarks/benchmark_base.py

```diff
@@ -64,6 +64,14 @@ def tardis_config_verysimple(self):
             YAMLLoader,
         )
 
+    @property
+    def config_rpacket_tracking(self):
+        config = Configuration.from_yaml(
+            "tardis/io/configuration/tests/data/tardis_configv1_verysimple.yml"
+        )
+        config.montecarlo.tracking.track_rpacket = True
```

Comment:
Just changing the track_rpacket to True and keeping everything else the same, so it can be checked how much the simulation takes extra with tracking enabled.

Reply:
Can you change this function decorator to `@functools.cached_property`? You will need to import functools library.

Reply:
Sure.